documentation/modules/upgrading/proc-upgrade-cluster-operator.adoc

```diff
@@ -41,15 +41,11 @@ Wait for the rolling updates to complete.
 +
 [source,shell,subs="+quotes,attributes"]
 ----
-kubectl get pod my-cluster-kafka-0 -o jsonpath='{.spec.containers[0].image}'
+kubectl get pods my-cluster-kafka-0 -o jsonpath='{.spec.containers[0].image}'
 ----
 +
 The image tag shows the new Strimzi version followed by the Kafka version. For example, `_NEW-STRIMZI-VERSION_-kafka-_CURRENT-KAFKA-VERSION_`.
```

Comment:
This would actually have only the Strimzi version in the tag. For example `quay.io/strimzi/operator:0.21.1`.

Reply:
Done. I added a new attribute for the example image tag: `quay.io/strimzi/operator:0.21.1`
Using an example works better here.